examples/example.krun

```diff
@@ -64,3 +64,7 @@ N_EXECUTIONS = 2  # Number of fresh processes.
 # You should set this high enough for the system to cool down a bit.
 # The default (if omitted) is 60 seconds.
 TEMP_READ_PAUSE = 1
+
+# Commands to run before and after each process execution
+PRE_EXECUTION_CMDS = ["date > /tmp/pre_exec"]
```

Comment:
Let's put in examples of pre/post commands people might want to execute (e.g. `/etc/rc.d/cron stop` or whatever).